internal/model/recommendation_set.go

```diff
@@ -96,12 +92,3 @@ func (r *RecommendationSet) CreateRecommendationSet() error {
 	return nil
 }
 
-func DeleteStaleRecommendationSet(workload_id uint, containers []string) error {
-	db := database.GetDB()
-	result := db.Where("workload_id = ? AND container_name NOT IN ?", workload_id, containers).Delete(&RecommendationSet{})
-	if result.Error != nil {
-		return result.Error
-	}
-
-	return nil
-}
```

Comment:
Why we are removing this?

Reply:
:thinking: , would it be okay to get rid of recommendations for non-existing containers from a historical data pov?

Reply:
From historical data pov we should keep it. I was just think from user perspective that it should not happen we are showing some recommendation but actually that container does not exist in customer environment. 
Anyway I think we can remove this method for now, we will take care of stale data in further iterations.